Lombiq.Analyzers.PowerShell/Rules/Measure-LineContinuation/Measure-LineContinuation.psm1

```diff
@@ -38,12 +38,12 @@ function Measure-LineContinuation
                     $PSItem.Kind -eq [System.Management.Automation.Language.TokenKind]::LineContinuation })
             {
                 $results += [Microsoft.Windows.Powershell.ScriptAnalyzer.Generic.DiagnosticRecord]@{
-                    "Extent" = $lineContinuationToken.Extent
-                    "Message" = 'Using backtick (line continuation) makes the code harder to read and' +
-                    ' maintain. Please consider using parameter splatting instead.'
```

Comment:
I would prefer the spaces at the end of the previous line. What do you think?

Reply:
I don't, because one of the possible cases where it ends in punctuation: If the following text is removed, then the space at the end wouldn't need to be there. So, IMO it's the following text's responsibility to define whether a space is needed or not.

Reply:
I see, makes sense, although it looks a bit weird 😏